examples_tests/61.BlitFilterTest/alpha_test/parameters.glsl

```diff
@@ -0,0 +1,9 @@
+#ifndef _NBL_GLSL_BLIT_ALPHA_TEST_PARAMETERS_INCLUDED_
+#define _NBL_GLSL_BLIT_ALPHA_TEST_PARAMETERS_INCLUDED_
+
+struct nbl_glsl_blit_alpha_test_parameters_t
+{
+	float referenceAlpha;
+};
+
+#endif
```

Comment:
delete the whole file, lets unify the push constants into a single struct

Reply:
Resolved.